src/network/networkd-network.h

```diff
@@ -95,6 +95,7 @@ struct Network {
 
         char *description;
 
+        NetDev *batadv;
```

Comment:
Not used?

Reply:
It seems to crash when not defined. I'll check

Reply:
Hmm??